### PR TITLE
Add more compliance info to onboarding guide and checklist

### PR DESCRIPTION
**Pull request by brittag**
This PR adds onboarding requirements for our compliance needs, but I tried to smooth out the overall onboarding flow at the same time so that I'm not making the onboarding experience _worse_ overall, hopefully!

Onboarding guide changes:

* Update intro to reflect our current business context.
* Speak directly to the reader as a newcomer.
* Move glossary to the bottom of the page, since it's suitable for skimming rather than thorough reading.
* Move "check out the team" item to the checklist.
* Add a section for security/compliance context, including [IA-2 (11)](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=IA-2#enhancement-11) note.

Onboarding checklist changes:

* Update docs links.
* Reorder first items.
* Add more security reading to required list.

cc @mogul

### CODE REVIEW

File: Onboarding.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Joining the cloud.gov team
 
-cloud.gov is the platform that hosts the applications 18F builds with and for other agencies. cloud.gov helps 18F teams attack core impediments to smooth, iterative deployment of government services, like compliance, security and scalability. The cloud.gov team operates, supports, and develops cloud.gov so that we can offer it to agencies directly as a service they can use on their own.
+cloud.gov helps government teams attack core impediments to smooth, iterative deployment of government services, including compliance, security and scalability. The cloud.gov team operates, supports, and develops cloud.gov so that we can offer it to agencies directly as a service they can use on their own.
```
Comment (adborden) [inline Onboarding.md]: Second sentence uses a serial comma, the first does not, but I'm not sure which is our preference :)

File: OnboardingChecklist.md [hunk 2]
```diff
@@ -26,13 +26,14 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
-- [ ] Bookmark the [pertinent links listed here](https://github.com/18F/cg-product/blob/master/PertinentLinks.md).
 - [ ] Figure out who your onboarding buddy is (they should reach out to you) and make sure this issue is assigned to them
 - [ ] Read [the team onboarding document](https://github.com/18F/cg-product/blob/master/Onboarding.md) for more context about cloud.gov
+- [ ] Check out the [cloud.gov team roster sheet](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit#gid=0) to get a sense of who's working on cloud.gov and in what capacity. (Note that we work closely with the [18F Infrastructure](https://github.com/18F/infrastructure) team, but they're not considered part of the cloud.gov team.)
+- [ ] Bookmark the [pertinent links listed here](https://github.com/18F/cg-product/blob/master/PertinentLinks.md).
```
Comment (adborden) [inline OnboardingChecklist.md]: There's a period at the end of this phrase which is inconsistent with the other bullet points.

File: Onboarding.md (hunk 1 above)
Comment (brittag) [inline Onboarding.md]: Good catch! 18F style is to use serial comma - https://pages.18f.gov/content-guide/punctuation/#commas

File: OnboardingChecklist.md (hunk 2 above)
Comment (brittag) [inline OnboardingChecklist.md]: I'll fix! 18F style is to use periods when bullet points are complete sentences (https://pages.18f.gov/content-guide/punctuation/#bulleted-lists), and these are complete imperative sentences.

File: OnboardingChecklist.md [hunk 3]
```diff
@@ -26,100 +26,105 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
+- [ ] Read the [January 2016 cloud.gov "Executive Business Case" document](https://docs.google.com/document/d/138OcG0Lt6gr9J0wM0TzzPNyTROmYAwfLIDujtweiwGw/edit#) for greater context about cloud.gov's potential impact in government.
+- [ ] [Deploy a sample application to cloud.gov](https://cloud.gov/docs/getting-started/your-first-deploy/) to get familiar with the basics of the PaaS from a user's perspective.
+- [ ] Check out the [roadmap](https://18f.aha.io/products/CGP/feature_cards) to get a high-level view of recently-completed, in-progress, and upcoming features.
+- [ ] Install the [ZenHub](https://zenhub.io) browser extension to enable you to see kanban boards, hierarchical issues, and burndown charts in GitHub.
+- [ ] Read the [Delivery Process document](https://github.com/18F/cg-product/blob/master/DeliveryProcess.md) to learn about how we work.
+- [ ] Join [the cloud.gov support Google Group](https://groups.google.com/a/gsa.gov/forum/?hl=en#!forum/cloud-gov-support), so you can see how we handle non-18F support and get visibility of technical discussions happening with outside groups.
```
Comment (mogul) [inline OnboardingChecklist.md]: We're deprecating individual membership in the list now... This needs to be replaced with something about provisioning access to Zendesk.

File: OnboardingChecklist.md [hunk 4]
```diff
@@ -26,100 +26,105 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
-- [ ] Subscribe (through the GitHub watch function) to the [cg-site](https://github.com/18F/cg-site) GitHub repository notifications
+- [ ] Read the [Configuration Management Plan](https://cloud.gov/docs/ops/configuration-management/).
+- [ ] Read our [sharing secret keys](https://cloud.gov/docs/ops/secrets/#sharing-secret-keys) policy.
+- [ ] Review the [18F requirements for password management](https://handbook.18f.gov/password-requirements/).
+- [ ] Review the [18F open source policy guidance about protecting sensitive information](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information).
+- [ ] Coordinate with your onboarding buddy to go through [nonpublic information training](https://docs.google.com/presentation/d/1uB4MlGCu8ZYUxjKVZKwicQ95MvLxaT4Mh93y6w79GPw/edit#slide=id.p) within 60 days of joining the team (and annually after that).
+- [ ] Subscribe to [the cloud.gov team calendar](https://calendar.google.com/calendar/embed?src=gsa.gov_0samf7guodi7o2jhdp0ec99aks@group.calendar.google.com&ctz=America/Los_Angeles) (click the + in the bottom right) so you know when assorted team meetings are happening in the various streams.
```
Comment (mogul) [inline OnboardingChecklist.md]: s/streams/squads/

File: OnboardingChecklist.md [hunk 5]
```diff
@@ -26,100 +26,105 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
-- [ ] Review the main [front end board](https://github.com/18F/cg-dashboard/pulls#boards?repos=55727091,39210774,49169967,40567233&labels=Navigator&showPRs=false) (ensure to filter by the "Navigator" label)
-- [ ] Bookmark link to [design folder](https://drive.google.com/drive/u/1/folders/0BwLqM4Nicmq-bUt0NjRjclFMUEU)
-- [ ] Review the primary cloud.gov sites: [the dashboard](https://dashboard.cloud.gov/#/), [main landing page](https://cloud.gov/), and [documentation](https://docs.cloud.gov/).
-- [ ] [Request access to 18F Google Analytics](https://handbook.18f.gov/google-analytics/), so you can view cloud.gov site analytics ([including for the dashboard](https://docs.google.com/document/d/1gSbP2ak2a3QLpCZIF_KlbQ2QHE6RjDI-7ZnnrJZvMDE/edit))
-- [ ] Ask for an invite to a DigitalGov Search account for cg-docs, so you can configure it and view analytics
+- [ ] Join the `#cloud-gov-navigator` channel on Slack.
+- [ ] Ping @standup-bot for instructions on front end channel standup.
+- [ ] Review the main [front end board](https://github.com/18F/cg-dashboard/pulls#boards?repos=55727091,39210774,49169967,40567233&labels=Navigator&showPRs=false) (filter by the "Navigator" label).
```
Comment (mogul) [inline OnboardingChecklist.md]: This should omit the specific repositories so the URL is forward-compatible, using just the `Navigator` label to select the proper view.

File: OnboardingChecklist.md [hunk 6]
```diff
@@ -26,100 +26,105 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
-- [ ] Set up the [landing/docs page](https://github.com/18F/cg-site), [dashboard](https://github.com/18F/cg-dashboard), and [cg-style](https://github.com/18F/cg-style) locally
-- [ ] Set up cg-style to be [linked to the other sites locally](https://github.com/18F/cg-style#development-and-contributing-setup)
-- [ ] Have cloud.gov person send the cg-dashboard testing env vars through Fugacious
+- [ ] Review [dashboard contributing guide](https://github.com/18F/cg-dashboard/blob/master/CONTRIBUTING.md) and [cg-style standards](https://github.com/18F/cg-style/blob/master/documentation/frontend_standards.md) which help frame what we're looking for in code review.
+- [ ] Set up the [landing/docs page](https://github.com/18F/cg-site), [dashboard](https://github.com/18F/cg-dashboard), and [cg-style](https://github.com/18F/cg-style) locally.
+- [ ] Set up cg-style to be [linked to the other sites locally](https://github.com/18F/cg-style#development-and-contributing-setup).
+- [ ] Have cloud.gov person send the cg-dashboard testing env vars through Fugacious.
```
Comment (mogul) [inline OnboardingChecklist.md]: s/env vars/environment variables/, s/cg-dashboard/`cg-dashboard`/

File: OnboardingChecklist.md [hunk 7]
```diff
@@ -26,100 +26,105 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
-- [ ] Set up the [landing/docs page](https://github.com/18F/cg-site), [dashboard](https://github.com/18F/cg-dashboard), and [cg-style](https://github.com/18F/cg-style) locally
-- [ ] Set up cg-style to be [linked to the other sites locally](https://github.com/18F/cg-style#development-and-contributing-setup)
-- [ ] Have cloud.gov person send the cg-dashboard testing env vars through Fugacious
+- [ ] Review [dashboard contributing guide](https://github.com/18F/cg-dashboard/blob/master/CONTRIBUTING.md) and [cg-style standards](https://github.com/18F/cg-style/blob/master/documentation/frontend_standards.md) which help frame what we're looking for in code review.
+- [ ] Set up the [landing/docs page](https://github.com/18F/cg-site), [dashboard](https://github.com/18F/cg-dashboard), and [cg-style](https://github.com/18F/cg-style) locally.
+- [ ] Set up cg-style to be [linked to the other sites locally](https://github.com/18F/cg-style#development-and-contributing-setup).
```
Comment (mogul) [inline OnboardingChecklist.md]: s/cg-style/`cg-style`/

**Pull request merged**